index.js

```diff
@@ -43,6 +43,95 @@
+//.       // ...
+//.     } else {
+//.       return f(R.head(xs));
+//.     }
+//.
+//. Using the Maybe type renders such guards (and null checks) unnecessary.
+//. Changing functions such as `R.head` to return Maybe values was proposed
+//. in [ramda/ramda#683][], but was considered too much of a stretch for
+//. JavaScript programmers. Sanctuary was released the following month,
+//. in January 2015, as a companion library to Ramda.
+//.
+//. In addition to broadening in scope in the years since its release,
+//. Sanctuary's philosophy has diverged from Ramda's in several respects.
+//.
+//. ### Totality
+//.
+//. Every Sanctuary function is defined for every value which is a member of
+//. the function's input type. Such functions are known as [total functions][].
+//. Ramda, on the other hand, contains a number of [partial functions][].
+//.
+//. ### Information preservation
+//.
+//. Certain Sanctuary functions preserve more information than their Ramda
+//. counterparts. For example, `S.tail(['foo'])` and `S.tail([])` produce
+//. different results, whereas `R.tail(['foo'])` and `R.tail([])` produce the
+//. same result.
+//.
+//. ### Invariants
+//.
+//. Sanctuary performs rigorous [type checking][] of inputs and outputs, and
+//. throws a descriptive error if a type error is encountered. This allows bugs
+//. to be caught and fixed early in the development cycle.
+//.
+//. Ramda operates on the [garbage in, garbage out][GIGO] principal. Functions
+//. are documented to take arguments of particular types, but these invariants
+//. are not enforced. The problem with this approach in a language as
+//. permissive as JavaScript is that there's no guarantee that garbage input
+//. will produce garbage output ([ramda/ramda#1413][]). Ramda performs ad hoc
+//. type checking in some such cases ([ramda/ramda#1419][]).
+//.
+//. Sanctuary can be configured to operate in garbage in, garbage out mode.
+//. Ramda cannot be configured to enforce its invariants.
+//.
+//. ### Variadic functions
+//.
+//. Ramda provides several functions with take any number of arguments. These
+//. are known as [variadic functions][]. Additionally, Ramda provides several
+//. functions which take variadic functions as arguments. Although natural in
+//. a dynamically typed language, variadic functions are at odds with the type
+//. notation Ramda and Sanctuary both use, leading to some indecipherable type
+//. signatures such as this one:
+//.
+//.     R.lift :: (*... -> *...) -> ([*]... -> [*])
+//.
+//. Sanctuary has no variadic functions, nor any functions which take variadic
+//. functions as arguments. Sanctuary provides two "lift" functions, each with
+//. a helpful type signature:
+//.
+//.     S.lift2 :: Apply f => (a -> b -> c) -> f a -> f b -> f c
+//.     S.lift3 :: Apply f => (a -> b -> c -> d) -> f a -> f b -> f c -> f d
+//.
+//. ### Implicit context
+//.
+//. Ramda provides a handful of functions for working with `this`. Sanctuary
```

Comment:
[`R.bind`][1] and [`R.invoke`][2] spring to mind. There may be others which I'm forgetting.


[1]: http://ramdajs.com/docs/#bind
[2]: http://ramdajs.com/docs/#invoker